pkg/asset/manifests/openshift.go

```diff
@@ -45,6 +47,12 @@ type Openshift struct {
 	FileList []*asset.File
 }
 
+// Used for templating the baremetal CR.
+type baremetalTemplateData struct {
```

Comment:
why not move it closer to https://github.com/openshift/installer/blob/4bb7986830ec15de43f6f0b6a5f40d9b761925bf/pkg/asset/manifests/template.go#L77

or however we did the platform separation for the bootkube/bootstrap.ign templating.